### ISSUE TITLE
RAG/retriever tool not working for both RAG and assistant

### ISSUE THREAD

**Issue by weipienlee** [langchain-ai/opengpts#259]
Tried txt and pdf files smaller than 10MB. In all cases, at first retrieval action i get this error:

`File "/usr/local/lib/python3.11/site-packages/langchain_community/vectorstores/pgvector.py", line 656, in _handle_field_filter raise ValueError(
ValueError: Invalid operator: in. Expected one of {'$between', '$in', '$ne', '$lte', '$ilike', '$gt', '$eq', '$or', '$and', '$gte', '$lt', '$like', '$nin'}`

Extra quirk after assistant creation, the retrieval tool has vanished from "save" assistant. However, with the right question the retrieval will still initiate and fail.

fyi - running with docker on Apple silicon

**Comment by akira** [comment]
I believe this was fixed in this PR: https://github.com/langchain-ai/opengpts/pull/252 - have you tried the latest version?

**Comment by weipienlee** [comment]
Yes, it was literally what the error said:). Tnx for the headsup @akira!

**Comment by weipienlee** [comment]
Do I need to need to close this myself? Let me try

**Comment by mkorpela** [comment]
Thanks for the issue @weipienlee. Seems you could not close it => closing.
